armadillo/src/main/java/at/favre/lib/armadillo/OnSecurePreferenceChangeListener.java

```diff
@@ -0,0 +1,34 @@
+package at.favre.lib.armadillo;
+
+import android.content.SharedPreferences;
+import android.support.annotation.NonNull;
+
+/**
+ * Allow to register preference change listeners that receive
+ * @since 03.07.2019
```

Comment:
Ok, for this one I just applied some of the existing code patterns which used dates. Do you prefer if I remove the @since tag, as I don't actually know next release version. Or either you tell me if it's OK to say it will be `0.9.0`?

Reply:
You can set 0.9.0 or just remove it, whichever you prefer.